tests/integration/servicemesh/main_test.go

```diff
@@ -0,0 +1,40 @@
+//go:build integ
+// +build integ
+
+// Copyright Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicemesh
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/tests/integration/servicemesh/maistra"
+)
+
+var i istio.Instance
+
+func TestMain(m *testing.M) {
+	// do not change order of setup functions
+	framework.
+		NewSuite(m).
+		RequireMaxClusters(1).
+		Setup(maistra.ApplyGatewayAPICRDs).
```

Comment:
It was necessary to apply Gateway API CRD, because otherwise CRD client is failing when `--enableCRDScan=false` is set. Btw. does OCP comes with Gateway API installed?

Reply:
I think we shoujld rather disable Gateway API. That's  what we do in the operator

Reply:
and no, I don't think OCP comes with Gateway API CRDs installed - which is why we should disable this by default. Reminds me.. we should really fix the CRD scan. There's a way to do it without privileges

Reply:
Ok, I didn't know about flags `PILOT_ENABLE_GATEWAY_API*`. I disabled them.